spec/factories/uk_new_secondary_completion.rb

```diff
@@ -0,0 +1,11 @@
+FactoryBot.define do
+  factory :uk_new_secondary_completion do
```

Comment:
what step/url does this reference ?

Reply:
https://github.com/DFE-Digital/get-teacher-training-adviser-service/blob/master/app/models/uk_new_secondary_completion.rb